detect_secrets/settings.py

```diff
@@ -221,7 +221,7 @@ def json(self) -> Dict[str, Any]:
         for plugin in get_plugins():
             # NOTE: We use the initialized plugin's JSON representation (rather than using
             # the configured settings) to deal with cases where plugins define their own
-            # default variables, that is not necessarily carried through through the
+            # default variables, that is not necessarily carried through the
```

Comment:
This is a fairly new heuristic, but I think it's right.

Reply:
LGTM